technische-universitat-dresden-medizin.csl

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Technische Universität Dresden - Medizin</title>
+    <title-short>TUD Medizin</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-medizin</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-medizin" rel="self"/>
+    <link href="https://www.zotero.org/styles/harvard1" rel="template"/>
+    <link href="http://tu-dresden.de/die_tu_dresden/fakultaeten/medizinische_fakultaet/forschung/die_tu_dresden/fakultaeten/medizinische_fakultaet/forschung/dokumente/dok_akadgrad/documents_newpromordnung/zitierrichtinien_fuer_die_dissertation_formblatt9.0" rel="documentation" xml:lang="DE-de"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>http://www.slub-dresden.de</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
+    <summary>Zitierstil entsprechend den Zitierrichtlinien für die Dissertation, Medizinische Fakultät Carl Gustav Carus, Technische Universität Dresden. Stil basiert auf dem Zitierstil der Zeitschrift Human Mutation</summary>
```

Comment:
The existent journal style is not thoroughly programmed. Also, the new style differs slightly from Human Mutation. It is an exact rendering of the style guide of TU Dresden, Faculty of Medicine.